www/source/javascripts/community.js

```diff
@@ -1,19 +1,128 @@
-var community_events = function(data) {
-    if ($(".community-events")) {
-        if (Array.isArray(data)) {
-            data.forEach(function(e) {
-                var ev = "<p>";
-                ev += e.title;
-                ev += "<br>";
-                ev += e.link;
-                ev += "<br>";
-                ev += e.pub_date;
-                ev += "<br>";
-                ev += e.description;
-                ev += "</p>";
-
-                $(".community-events").append(ev);
-            });
-        }
+$(function() {
+  var htmlDecode = function(input) {
```

Comment:
It looks like the only place we use this is for the `guid` field, which is just a string. Do we need it?